neurokit2/signal/signal_interpolate.py

```diff
@@ -3,7 +3,9 @@
 import scipy.interpolate
 
 
-def signal_interpolate(x_values, y_values, x_new=None, method="quadratic"):
```

Comment:
Since there's a separate PR for the changes to `signal_interpolate()` now, could you revert all changes to this file (including formatting) such that it doesn't show up in this PR?